docs/bnb_smart_chain/slashing/slash-events.md

```diff
@@ -0,0 +1,50 @@
+# BSC Slashing Events
```

Comment:
The title and the sidebar name is "slash event", I can hardly understand what is slash event and what this section is going to talk.  I think "Slash Rules" is better, because it is talking about the detailed slash BNB amount and params.

Reply:
fixed